contribs/emissions/src/main/java/org/matsim/contrib/emissions/WarmEmissionAnalysisModule.java

```diff
@@ -307,22 +304,19 @@ void throwWarmEmissionEvent(double leaveTime, Id<Link> linkId, Id<Vehicle> vehic
 
 				// compute faction.  This cannot be done earlier since efkey.component is needed.
 				fractionStopGo = getFractionStopAndGo(freeVelocity_ms * 3.6, averageSpeed_kmh, vehicleInformationTuple, efkey);
-				logger.info("fractionStopGo is: " + fractionStopGo);
 
 				double efStopGo_gpkm = 0.;
 				if (fractionStopGo > 0) {
 					// compute emissions from stop-go fraction:
 					efkey.setTrafficSituation(STOPANDGO);
 					efStopGo_gpkm = getEf(vehicleInformationTuple, efkey).getFactor();
-					logger.warn("pollutant=" + warmPollutant + "; efStopGo=" + efStopGo_gpkm);
```

Comment:
I would prefer setting it to logger.debug(...) instead of removing it.
Then it is available if one would like to have it.

Reply:
Okay, I can do that. I find the emission contrib's logging quite verbose overall anyway, though.